src/main/java/com/conveyal/datatools/manager/jobs/validation/SharedStopsValidator.java

```diff
@@ -24,24 +23,30 @@ public class SharedStopsValidator extends FeedValidator {
     private static final Logger LOG = LoggerFactory.getLogger(SharedStopsValidator.class);
 
     Feed feed;
+    String feedId;
     Project project;
 
-    public SharedStopsValidator(Project project) {
+    public SharedStopsValidator(Project project, String feedId) {
         super(null, null);
         this.project = project;
+        this.feedId = feedId;
     }
 
     // This method can only be run on a SharedStopsValidator that has been set up with a project only
     public SharedStopsValidator buildSharedStopsValidator(Feed feed, SQLErrorStorage errorStorage) {
         if (this.project == null) {
             throw new RuntimeException("Shared stops validator can not be called because no project has been set!");
         }
-        return new SharedStopsValidator(feed, errorStorage, this.project);
+        if (this.feedId == null) {
+            throw new RuntimeException("Shared stops validator can not be called because no feed ID has been set!");
+        }
+        return new SharedStopsValidator(feed, errorStorage, this.project, this.feedId);
     }
-    public SharedStopsValidator(Feed feed, SQLErrorStorage errorStorage, Project project) {
+    public SharedStopsValidator(Feed feed, SQLErrorStorage errorStorage, Project project, String feedId) {
```

Comment:
The other constructor already defines the `project` and `feedId`. I don't think it is needed here.

Reply:
This is the hack we need to use to do the half-instantiation